llap-client/src/test/org/apache/hadoop/hive/llap/registry/impl/TestLlapZookeeperRegistryImpl.java

```diff
@@ -124,6 +126,17 @@ public void testUpdate() throws Exception {
             attributes.get(LlapRegistryService.LLAP_DAEMON_NUM_ENABLED_EXECUTORS));
   }
 
+  @Test
+  public void testPersistentNodePath() {
+
```

Comment:
minor nit, sorry, we usually don't put a blank line as the first line of a method (you'll have to push again as tests failed unfortunately, sigh)

Reply:
fixed